src/main/proto/wfa/virtual_people/common/event.proto

```diff
@@ -52,6 +55,9 @@ message UserInfo {
   // Creation or initialization time of the user id, in microseconds
   // since epoch.
   int64 creation_time_usec = 5;
+
+  // The fingerprint of user_id. Only used in Labeler.
```

Comment:
Maybe "Only set and used in Labeler." to make it clear this field is not set in input events.

Reply:
Done.